Submodels/MTSET_call.py

```
@@ -177,44 +209,50 @@ def FileGenerator(self,
         if self.process.poll() is not None:
             try:
                 self.process.wait(timeout=1)
+            
             except subprocess.TimeoutExpired:
                 self.process.kill()
                 raise OSError("Something went wrong in the MTSET call. \
                             MTSET was not closed following end of file generation. \
                             Run terminated.")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)